src/zope/app/apidoc/codemodule/browser/introspector.rst

```diff
@@ -27,20 +27,22 @@ and the name of the object:
 Of course, the root folder does not have a name. As you can see the type links
 directly to the API documentation of the class.
 
-The next section lists all directly provided interfaces. The root folder
-directly provides the ``ISite`` and ``IRootFolder`` interface, so we should
-see those:
+The next section lists all directly provided interfaces. The root
+folder directly provides the :class:`zope.site.interfaces.ISite` and
+:class:`zope.site.interfaces.IRootFolder` interface, so we should see
+those::
```

Comment:
So now I'm wondering what's the difference in Sphinx rendering regular doctest blocks (ReStructuredText [supports these natively](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#doctest-blocks)) and literal blocks that happen to contain doctests.

In other words, why are you adding `::` before each doctest block?